docs/source/common/trainer.rst

```diff
@@ -1808,3 +1809,19 @@ The metrics sent to the progress bar.
 
     progress_bar_metrics = trainer.progress_bar_metrics
     assert progress_bar_metrics["a_val"] == 2
+
+
+estimated_num_optimization_steps
+********************************
+
+Calculates the estimated number of optimization steps for the complete training. The value is inferred from DataLoaders, gradient
+accumulation factor and distributed setting.
+
+.. code-block:: python
```

Comment:
this seems the same as docstring later, so can we rather refer to the `estimated_num_optimization_steps` API?